example/ck_tile/17_grouped_gemm/grouped_gemm_tileloop.cpp

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2024-2025, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Our convention is  \<date of creation\>\[-\<date of last modification\>\]
```suggestion
// Copyright (c) 2025, Advanced Micro Devices, Inc. All rights reserved.
```